pyproject.toml

```diff
@@ -37,9 +37,13 @@ dependencies = [
     "liger-kernel>=0.5.4; platform_system != 'Darwin'",
 ]
 
+[tool.hatch.metadata]
+allow-direct-references = true
```

Comment:
Whats this change?

Reply:
The recent readme changes need this to launch experiments.